robottelo/api/utils.py

```diff
@@ -946,3 +946,26 @@ def skip_yum_update_during_provisioning(template=None, reverse=False):
         return update_provisioning_template(name=template, old=old, new=new)
     else:
         return update_provisioning_template(name=template, old=new, new=old)
+
+
+def hammer_api_timeout_set(timeout=-1, reverse=False):
+    """Set hammer API request timeout on Satellite
+
+    :param int timeout: request timeout in seconds
+    :param bool reverse: Reverses the request timeout
+    :return: Boolean True on success else exception
```

Comment:
ssh.command returns Boolean, I doubt here.

Reply:
that what i understood from is's behaviour and we are following same here too: 

https://github.com/SatelliteQE/robottelo/blob/445eea73d227f4f3f68de58eeab2cec43918463d/robottelo/api/utils.py#L936-L941

Reply:
That can be incorrect, take a look at  https://github.com/omkarkhatavkar/robottelo/blob/a55948b7a4575a72506c845e81e2d3049dbf5056/robottelo/ssh.py#L449 file we are ssh.command does not return Boolean

Reply:
good catch , could just add the 'ssh.command' in return

Reply:
done